hedera-node/hedera-mono-service/src/main/resources/bootstrap.properties

```diff
@@ -207,3 +207,4 @@ hedera.prefetch.threadPoolSize=4
 hedera.prefetch.codeCacheTtlSecs=600
 utilPrng.isEnabled=true
 tokens.autoCreations.isEnabled=true
+hedera.workflows.enabled=false
```

Comment:
We should probably add this to `hedera-mono-service/test/resource/standard.properties`, `test-clients/src/main/resource`.